usagov/vote-gov-drupal
# Vote-831: Hero Section Styling

## Jira ticket

[Vote-832](https://cm-jira.usa.gov/browse/VOTE-832)

## Description

Adding styles for Hero section of landing page.

## Deployment and testing

### Pre-deploy

1. Pull down branch and run `lando retune`

### Post-deploy

n/a

### QA/Test

1. Log in with `lando drush uli` and add and update the hero section to add heading, link and image 
2. Check that section matches mock up 

## Checklist for the Developer




## Checklist for the Peer Reviewers
